src/components/content/order/Services.tsx

```diff
@@ -31,18 +35,27 @@ function Services(): JSX.Element {
         }
         serviceVendorApi.listRegisteredServicesTree(categoryName).then((rsp) => {
             if (rsp.length > 0) {
-                let serviceList: { name: string; content: string; icon: string }[] = [];
+                let serviceList: { name: string; content: string; icon: string; defaultVersion: string }[] = [];
                 rsp.forEach((item) => {
                     let serviceItem = {
                         name: item.name || '',
                         content: item.versions[0].cloudProvider[0].details[0].description,
                         icon: item.versions[0].cloudProvider[0].details[0].icon,
+                        defaultVersion: item.versions.sort((v1, v2) => {
```

Comment:
can we refine the version sort? it seems v1.9 is newer than v.1.10 by this implementation.